pkg/skaffold/build/cluster/sources/sources.go

```diff
@@ -135,6 +138,23 @@ func podTemplate(clusterDetails *latest.ClusterDetails, artifact *latest.KanikoA
 	return pod
 }
 
+func setProxy(clusterDetails *latest.ClusterDetails, env []v1.EnvVar) []v1.EnvVar {
```

Comment:
Can we add tests for setProxy with following test cases
1. clusterDetails has only HTTPProxy set
2. clusterDetails has only HTTPSProxy set
3. clusterDetails has  HTTPProxy and HTTPSProxy set
4. clusterDetails has nothing set returns nothing

Reply:
Sure will do that.

Reply:
added